Makefile

```
@@ -191,92 +184,87 @@ test: ## Run Go unit tests
 
 ##@ Docker
 
+export DOCKER_REGISTRY DOCKER_TAG VERSION BUILD_TIMESTAMP COMMIT_HASH VMCLARITY_TOOLS_BASE
+
 .PHONY: docker
-docker: docker-apiserver docker-cli docker-orchestrator docker-ui docker-ui-backend docker-cr-discovery-server ## Build All Docker images
+docker: ## Build All Docker images
+	$(info Building all docker images ...)
+	docker buildx bake
+
+.PHONY: docker-e2e
+docker-e2e: ## Build Docker images for e2e tests
+	$(info Building docker images for e2e tests ...)
+	docker buildx bake e2e
 
 .PHONY: docker-apiserver
 docker-apiserver: ## Build API Server container image
 	$(info Building apiserver docker image ...)
-	docker build --file ./Dockerfile.apiserver --build-arg VERSION=$(VERSION) \
-		--build-arg BUILD_TIMESTAMP=$(BUILD_TIMESTAMP) \
-		--build-arg COMMIT_HASH=$(COMMIT_HASH) \
-		-t $(VMCLARITY_APISERVER_IMAGE) .
-
-ifneq ($(strip $(VMCLARITY_TOOLS_BASE)),)
-VMCLARITY_TOOLS_CLI_DOCKER_ARG=--build-arg VMCLARITY_TOOLS_BASE=${VMCLARITY_TOOLS_BASE}
-endif
+	docker buildx bake apiserver
```

`make docker-apiserver` returns `ERROR: failed to find target apiserver`
The proper name is `vmclarity-apiserver` as defined in the bake. Same should be updated for all other docker related make targets.
```suggestion
	docker buildx bake vmclarity-apiserver
```